lonestar/CMakeLists.txt

```diff
@@ -12,7 +12,7 @@ endfunction()
 
 function(add_test_scale type app)
   set(no_value_options NOT_QUICK NO_VERIFY)
-  set(single_value_options INPUT INPUT_URI)
+  set(single_value_options INPUT INPUT_URI REL_TOL MEAN_TOL)
```

Comment:
Why do these need to vary? I totally believe they do, but please document the specific reasons.